pathfinder/tests/test_blockchain_listener.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+from web3 import Web3
+
+from pathfinder.utils.blockchain_listener import BlockchainListener, create_channel_event_topics
+
+
+def test_blockchain_listener(
+    web3: Web3,
+    wait_for_blocks,
+    generate_raiden_clients,
+    blockchain_listener: BlockchainListener,
+    ethereum_tester,
+):
+    blockchain_listener.required_confirmations = 4
+    blockchain_listener.start()
+    blockchain_listener.wait_sync()
+
+    unconfirmed_channel_open_events = []
+    confirmed_channel_open_events = []
+
+    blockchain_listener.add_unconfirmed_listener(
+        create_channel_event_topics(),
+        lambda e: unconfirmed_channel_open_events.append(e),
```

Comment:
Why not just
`unconfirmed_channel_open_events.append`
instead of
`lambda e: unconfirmed_channel_open_events.append(e)`?

Reply:
Good point, not sure why I did this in the first place -.-